src/test/java/org/openrewrite/java/migrate/jakarta/UpdateApacheWSSecurityPackagesTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.jakarta;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.config.Environment;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class UpdateApacheWSSecurityPackagesTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion().
+            classpathFromResources(new InMemoryExecutionContext(), "wss4j-1.6.19", "wss4j-ws-security-common-2.0.0")).
+          recipe(Environment.builder().scanRuntimeClasspath("org.openrewrite.java.migrate.jakarta")
+            .build()
+            .activateRecipes("org.openrewrite.java.migrate.jakarta.UpdateApacheWSSecurityPackages"));
+    }
+    @Test
+    public void updateApache(){
+        rewriteRun(
+          //language=java
+          java("""
+            package com.test;
+
+            import org.apache.ws.security.WSSecurityException;
+            import org.apache.ws.security.components.crypto.CryptoBase;
+            public class Test {
+                public static void main(String[] args)  {
+            
+                    WSSecurityException x = new WSSecurityException(1,"");
+            
+                    CryptoBase base = null;
+            
+                }
+            }
+            """, """
+            package com.test;
+            
+            import org.apache.wss4j.common.ext.WSSecurityException;
+            import org.apache.wss4j.common.ext.components.crypto.CryptoBase;
+            public class Test {
+                public static void main(String[] args)  {
+            
+                    WSSecurityException x = new WSSecurityException(1,"");
+            
+                    CryptoBase base = null;
+            
+                }
+            }
+            """));
```

Comment:
Can you scrub your tests of anything that's not needed, and format them to conform to the other tests we have?
- text blocks start on a new line
- no package declaration when not necessary
- no unnecessary blank lines
- opening and closing braces each on a new line
- no unused method modifiers (`public static`)
- or unused arguments `(String[] args)`

Having a consistent style helps readability, and not having to read unnecessary elements helps focus on what matters only. Otherwise it might seem something is significant ( such as here `public static void main(String[] args)`), when in reality it's not used at all.

Ideally we automate a lot of this type of feedback, but until then I'd prefer not to have to update these by hand myself. 

```suggestion
          java(
            """
            import org.apache.ws.security.WSSecurityException;
            import org.apache.ws.security.components.crypto.CryptoBase;
            class Foo {
                void bar()  {
                    WSSecurityException x = new WSSecurityException(1,"");
                    CryptoBase base = null;
                }
            }
            """,
            """
            import org.apache.wss4j.common.ext.WSSecurityException;
            import org.apache.wss4j.common.ext.components.crypto.CryptoBase;
            class Foo {
                void bar()  {
                    WSSecurityException x = new WSSecurityException(1,"");
                    CryptoBase base = null;
                }
            }
            """
          )
        );
```

Reply:
This also applies to the other tests in this PR.

Reply:
Done